src/components/ConstructionBanner/ConstructionBanner.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import Container from 'react-bootstrap/Container';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+function ConstructionBanner() {
+  return (
+    <div>
+      <Container fluid className="p1">
```

Comment:
Why did you decide to name those classes p1 and p2, and what styles are attached to them?